make image doesn't work on linux

I got following error on a linux box

```
[wwei@ip-172-31-15-90 yunikorn-k8shim]$ make image
mkdir -p _output/bin
building binary for scheduler docker image
GOOS=linux GOARCH=amd64 \
go build -a -o=_output/bin/k8s_yunikorn_scheduler -ldflags \
'-extldflags "-static" -X main.version=latest -X main.date=2019-07-18T08:51:44+0000' \
-tags netgo -installsuffix netgo \
./pkg/scheduler/
building scheduler docker image
sed: -e expression #1, char 1: unknown command: `.'
make: *** [image] Error 1
```
it works fine on Mac.

--- a/Makefile
+++ b/Makefile
@@ -76,7 +76,7 @@ build_image: init
 image: build_image
 	@echo "building scheduler docker image"
 	@cp ${RELEASE_BIN_DIR}/${BINARY} ./deployments/image/configmap
-	@sed -i '.bkp' 's/clusterVersion=.*"/clusterVersion=${VERSION}"/' deployments/image/configmap/Dockerfile
+	@sed -i'.bkp' 's/clusterVersion=.*"/clusterVersion=${VERSION}"/' deployments/image/configmap/Dockerfile
 	@coreSHA=$$(go list -m "github.com/cloudera/yunikorn-core" | cut -d "-" -f4) ; \
 	siSHA=$$(go list -m "github.com/cloudera/yunikorn-scheduler-interface" | cut -d "-" -f5) ; \
 	shimSHA=$$(git rev-parse --short=12 HEAD) ; \

